infer/src/integration/Driver.ml

```diff
@@ -390,7 +380,12 @@ let mode_of_build_command build_cmd (buck_mode : BuckMode.t option) =
       if not (List.is_empty Config.clang_compilation_dbs) then (
         assert_supported_mode `Clang "clang compilation database" ;
         ClangCompilationDB {db_files= Config.clang_compilation_dbs} )
-      else Analyze
+      else (
+        match (Config.cfg_json, Config.tenv_json) with
+        | Some cfg_json, Some tenv_json ->
+            JsonSIL {json_cfg= cfg_json; json_tenv= tenv_json}
+        | _, _ ->
```

Comment:
you can also write just `| _ ->` for that case

Reply:
Done